zipkin-lens/package.json

```diff
@@ -43,33 +43,25 @@
     "@types/redux-mock-store": "^1.0.2",
     "@types/shortid": "0.0.32",
     "@types/styled-components": "^5.1.0",
-    "@typescript-eslint/eslint-plugin": "^2.23.0",
-    "@typescript-eslint/parser": "^2.23.0",
     "classnames": "2.3.2",
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.7.0",
-    "eslint-config-airbnb-typescript": "^7.0.0",
-    "eslint-config-prettier": "^6.10.0",
-    "eslint-plugin-import": "^2.16.0",
-    "eslint-plugin-jest": "^23.8.2",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-react": "^7.12.4",
-    "eslint-plugin-react-hooks": "^2.5.0",
```

Comment:
These ESLint plugins are included in [react-scripts eslint config](https://github.com/facebook/create-react-app/tree/main/packages/eslint-config-react-app).